### DISCUSSION TITLE
How to concat a string to pass to env

### DISCUSSION THREAD

**Discussion by fullStackDataSolutions** [prisma/prisma#5994] [Q&A]
Currently I'm using:

```
///
datasource ipa {
  provider = "mysql"
  url      = env("DATABASE_URL")
}
```

I want to add a string for the schema name at the end of the environment variable for the scemas name as I am creating several clients for different schemas. 

How would I do something like this?

```
///
datasource ipa {
  provider = "mysql"
  url      = env(`${DATABASE_URL}/schemaName`)
}
```

I need to set it up in this manner because I am connecting to 10 schemas all in the same database, so it becomes too much to add 20 secrets to the cluster that are all the same exact the last value.

**Comment by tarazena** [comment]
why not append `/schema` to the connection url/env variable?
like
```
mysql://username:password@1.2.3.4:4444/db
```

**Reply by tarazena** [reply]
or based on Prisma docs
https://www.prisma.io/docs/concepts/components/prisma-schema#accessing-environment-variables-from-the-schema

**Reply by fullStackDataSolutions** [reply]
I'm already using this method, but will end up needing 10 or more environment variables that only differ slightly. I was hoping to have one that I could just add the schema too.

**Reply by ryands17** [reply]
@blazestudios23 👋 

You could use [this](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#datasources) method to dynamically change the schema and pass it to Prisma.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
